Documents/public-api.md

```diff
@@ -0,0 +1,36 @@
+# HMDA Platform Public API
+
+This documenatation describes de public HMDA Platform HTTP API
+
+## Institutions
+
+### Search
+
+* `/institutions?domain=<domain>`
+
+   * `GET` - Returns a list of institutions filtered by their email domain. If none are found, an HTTP 404 error code (not found) is returned
+
+   Example response, with HTTP code 200:
+
+   ```json
+   {
+     "institutions":
+     [
+        {
+          "id": "0",
+          "name": "Bank 0",
+          "domains": ["test@bank0.com"],
+          "externalIds":[
+            {
+              "name": "1234",
```

Comment:
`name` and `value` are switched.  I _think_ this happened when the parameters were renamed on https://github.com/cfpb/hmda-platform/pull/798/files#diff-1f02427f497c8439ae1bb2fbf7070c65R8.  I'm guessing when the data gets read in, it is assuming that order for the params?